src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -565,6 +565,163 @@ JSON example:
 }
 ```
 
+## Arterial Spin Labeling perfusion data
+
+Template:
+
+```Text
+sub-<label>/[ses-<label>/]
+    perf/
+       sub-<label>[_ses-<label>][_acq-<label>][_rec-<label>][_dir-<label>][_run-<index>]_asl.nii[.gz]
+       sub-<label>[_ses-<label>][_acq-<label>][_rec-<label>][_dir-<label>][_run-<index>]_asl.json
+       sub-<label>[_ses-<label>][_acq-<label>][_rec-<label>][_dir-<label>][_run-<index>]_aslcontext.tsv
+       sub-<label>[_ses-<label>][_acq-<label>][_rec-<label>][_dir-<label>][_run-<index>]_m0scan.nii[.gz]
+       sub-<label>[_ses-<label>][_acq-<label>][_rec-<label>][_dir-<label>][_run-<index>]_m0scan.json
+       sub-<label>[_ses-<label>][_acq-<label>][_rec-<label>][_run-<index>]_labeling.jpg
+```
+
+The complete ASL time series should be stored as a 4D NIfTI file in the original acquisition order, accompanied by two ancillary 
+files: `*_asl.json` and `*_aslcontext.tsv`. The `*_asl.nii.gz` and `*_m0scan.nii.gz` should contain appropriately scaled data and no scaling factors 
+should be stored in the ancillary files or in the NIfTI header. 
```

Comment:
I still have the same comment: https://github.com/bids-standard/bids-specification/pull/652/files#r520070205

Reply:
@gllmflndn I changed it now!

Reply:
Thank you!